cn/index.md

```diff
@@ -8,6 +8,14 @@ lang: cn
 
 Elixir 基础知识教程，受 Twitter 的 [Scala School](http://twitter.github.io/scala_school/) 启发编写。
 
+目前有多种语言版本： [Việt ngữ][vi], [汉语][cn], [Español][es], [日本語][jp] and [Português][pt].
```

Comment:
Per #270, we're thinking about removing theses languages from all indexes _except_ English.  This way you (nor other translators) need to keep track of need languages and updating your indexes.

  Any thoughts @cizixs?